web/views/project_view.ex

```diff
@@ -0,0 +1,11 @@
+defmodule CodeCorps.ProjectView do
+  use CodeCorps.Web, :view
+  use JaSerializer.PhoenixView
+
+  attributes [
+  	:slug, :title, :description, :icon_thumb_url, :icon_large_url,
+  	:long_description_body, :long_description_markdown,
+  	:inserted_at, :updated_at]
```

Comment:
What do you mean? Indentation?

If it's grouping, if I know there will be at least 3 lines, I prefer to group them in a more meaningful way, instead of just breaking at fixed points. In this case, common fields are on the same line.

* `:slug`, `:title`, and `:description` are on the same line
* `:icon_thumb_url`, `:icon_large_url` are on the same line

etc.